eisbuk-admin/src/components/atoms/OrganizationSettings/OrganizationSettings.tsx

```diff
@@ -0,0 +1,397 @@
+  };
+  const removeAdmin = (admin: string) => {
+    if (admin === currentUser) return;
+
+    const filteredAdmins = admins.filter((a) => a !== admin);
+    setValue(filteredAdmins);
+  };
+
+  const addAdmin = () => {
+    if (admin === "") return;
+    const newAdmins = [...admins, admin];
+    setValue(newAdmins);
+    setAdmin("");
+  };
+
+  const updateAdmin =
+    (i: number) => (e: React.ChangeEvent<HTMLInputElement>) => {
+      const adminsCopy = [...admins];
+      adminsCopy[i] = e.target.value;
+      setValue(adminsCopy);
+    };
+
+  return (
+    <>
+      <div className={classes.fieldSection}>
+        {admins.map((admin, i) => (
+          <div
+            key={admin}
+            className={clsx({
+              [classes.adminFieldGroup]: enableEdit,
+              [classes.disabledAdminFieldGroup]: !enableEdit,
+            })}
+          >
+            <Field
+              name={`admins[${i}]`}
+              onChange={updateAdmin(i)}
+              component={TextField}
+              value={admin}
+              variant={enableEdit ? "outlined" : "filled"}
+              disabled={!enableEdit || admin === currentUser}
+            />
+            {enableEdit && admin !== currentUser && (
+              <IconButton
+                type="button"
+                className={classes.closeButton}
+                onClick={() => removeAdmin(admin)}
+              >
+                <Cancel />
+              </IconButton>
+            )}
+          </div>
+        ))}
+      </div>
+      {enableEdit && (
+        <div className={classes.addAdmin}>
+          <Field
+            name={`newAdmin`}
+            component={TextField}
+            onChange={handleSetAdmin}
+            value={admin}
+            placeholder="Add admin"
+            variant={enableEdit ? "outlined" : "filled"}
+          />
+
+          <Button
+            onClick={addAdmin}
+            color="primary"
+            variant="contained"
+            className={classes.addAdminButton}
+          >
+            Add
+          </Button>
+        </div>
+      )}
+    </>
+  );
+};
+
+// #region styles
+const useStyles = makeStyles((theme) => ({
+  field: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+    marginRight: theme.spacing(2),
+    width: "21rem",
+  },
+  fieldSection: {
+    display: "flex",
+    flexFlow: "wrap",
+  },
+  sectionTitle: {
+    letterSpacing: 1,
+    fontSize: theme.typography.pxToRem(18),
+    fontFamily: theme.typography.fontFamily,
+    color: theme.palette.primary.light,
+  },
+  addAdminButton: {
+    marginLeft: theme.spacing(3),
+    borderRadius: theme.spacing(100),
+  },
+  adminFieldGroup: {
+    display: "flex",
+    // marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    width: "12.5rem",
+  },
+  disabledAdminFieldGroup: {
+    display: "flex",
+    // marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1),
```

Comment:
Remove the commented out properties if they're not to be used (and they don't seem needed)